Regression/PrimevalTitmouse/Regression.cs

```diff
@@ -321,7 +321,7 @@ private void ReceiveMouseChanged(object sender, ButtonPressedEventArgs e)
             }
 
             //Handle a Right Click
-            if (e.Button == SButton.MouseRight)
+            if (e.Button == SButton.MouseLeft)
             {
                 //If Right click is already being interpreted by another event (or we otherwise wouldn't process such an event. Ignore it.
                 if ((Game1.dialogueUp || Game1.currentMinigame != null || (Game1.eventUp || Game1.activeClickableMenu != null) || Game1.menuUp || Game1.fadeToBlack) || (who.isRidingHorse() || !who.canMove || (Game1.player.isEating || who.canOnlyWalk) || who.FarmerSprite.pauseForSingleAnimation))
```

Comment:
If we are switching to Shift+Left Click, are these conditions still valid/needed?

Reply:
What do you mean by switching to Shift+Left Click?